### ISSUE TITLE
Rails/FilePath possible regression

### ISSUE THREAD

**Issue by npetkov** [bbatsov/rubocop#5553]
Hi there,

I switched to rubocop from [master](https://github.com/bbatsov/rubocop/commit/fea6e01fc282597e3a6528c6de35307a8e4e3435) recently in order to make use of [an unreleased improvement](https://github.com/bbatsov/rubocop/issues/5448). It appears that some regression has creeped in that causes `Rails/FilePath` to report false positives.

--------

## Expected behavior

When running rubocop against a file containing interpolated strings, no `Rails/FilePath` warnings should be emitted for these strings.

## Actual behavior

Running rubocop on a Rails gem file scaffold yields:

```
Gemfile:4:15: C: Rails/FilePath: Please use Rails.root.join('path', 'to') instead.
  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?('/')
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

As can be seen from the below example, any interpolated string emits a false positive:

```
app/models/concerns/adjustable.rb:26:19: C: Rails/FilePath: Please use Rails.root.join('path', 'to') instead.
    define_method("#{state_name}?".to_sym) { state == STATES[state_name] }
                  ^^^^^^^^^^^^^^^^
```

## Steps to reproduce the problem

Run rubocop on any file containing interpolated strings.

## RuboCop version

0.52.1 (using Parser 2.4.0.2, running on ruby 2.5.0 x86_64-darwin17)

**Comment by koic** [comment]
Thanks for opening issue and using edge RuboCop! I reproduced it and fixed by #5556.
